src/core/core.config.js

```diff
@@ -31,20 +37,36 @@ function axisFromPosition(position) {
   }
 }
 
-export function determineAxis(id, scaleOptions) {
-  if (id === 'x' || id === 'y' || id === 'r') {
+export function determineAxis(id, ...scaleOptions) {
+  if (axisFromId(id)) {
     return id;
```

Comment:
> This might need to be the return value from axisFromId

Anyway I think you're right. It's better to use the return value. Seems to me more readable today.

Reply:
> Actually, I see how this works. I think naming the function `idMatchesAxis` or something similar might be clearer

Done! Used `idMatchesAxis`